Pu/issue617 (#619)
handsontable/hyperformula: CHANGELOG.md

### Changed
- GPU.js constructor needs to be provided directly to engine configuration. (#355)

> - A `'` symbol on the input is preserved in the value for serialization. (#617)

Author: ```suggestion
- An input value should be preserved through serialization more precisely. (#617)
```

> - A **breaking change**: `'` symbol on the input is preserved in the value. (#617)

Reviewer: It's a bug fix IMO. Of course the changed results are a breaking change, but we should first decide if the results should change at all.

Author: It changes the semantic that we agreed upon. For me it is kind of breaking.

> - A **breaking change**: `'` symbol on the input is preserved in the value. (#617)

Author: ```suggestion
- A `'` symbol on the input is preserved in the value for serialization. (#617)
```
